upgrading/automated_upgrades.adoc

```diff
@@ -732,6 +732,26 @@ other nodes in their class (`app` versus `infra`), one at a time.
 
 . Upgrade the {product-title} nodes running GlusterFS one at a time.
 
+.. Add the following parameters in the inventory file at *_/etc/ansible/hosts_*:
++
+----
+openshift_hosted_registry_storage_kind=glusterfs
+openshift_storage_glusterfs_heketi_image=registry.access.redhat.com/rhgs3/rhgs-volmanager-rhel7:<your_cns_vesion><1>
+openshift_storage_glusterfs_image=registry.access.redhat.com/rhgs3/rhgs-server-rhel7:<your_cns_vesion><1>
+openshift_storage_glusterfs_block_image=registry.access.redhat.com/rhgs3/rhgs-gluster-block-prov-rhel7:<your_cns_vesion><1>
+openshift_storage_glusterfs_s3_image=registry.access.redhat.com/rhgs3/rhgs-s3-server-rhel7:<your_cns_vesion><1>
+----
+<1> Specify the CNS version, such as *v3.9* or *v3.11.3*.
++
+.. Update the image tags from *latest* to `<your_cns_version>`, such as *v3.9* or *v3.11.3*, on the following resources, if they are present in your configuration:
++
+----
+# oc edit -n <glusterfs_namespace> ds glusterfs-<name>
```

Comment:
The rest of this procedure uses "$" as the prompt. Can these be updated to do the same, or do these really need to be run as root?

Reply:
Iran into a similar situation where I was instructed to use commands from a KCS article, where they used the root (`#`) prompt. In the case of that command (`oc debug`), you get root access in a running pod/container. (See conversation thread and resolution [here](https://github.com/openshift/openshift-docs/pull/28440#discussion_r595505126).)
@qinpingli I suspect that `$` is all that is needed here for `oc edit`, correct?

Reply:
We don't need to run `oc` cmd with the `root` user of linux os.  "$" is enough and a good habit to use linux os.

Reply:
Good catch!
Others lgtm!